feature/experimental/p4rt/otg_tests/google_discovery_protocol_packetout_test/google_discovery_protocol_packetout_test.go

```diff
@@ -179,9 +186,13 @@ func testPacketOut(ctx context.Context, t *testing.T, args *testArgs) {
 			t.Logf("Received %v packets on ATE port %s", counter1-counter0, port)
 
 			if test.expectPass {
+				rxVlanPackets := processCapture(t, args.ate, port)
 				if counter1-counter0 < uint64(packetCount*0.95) {
 					t.Fatalf("Not all the packets are received.")
 				}
+				if counter1 != uint64(rxVlanPackets) {
```

Comment:
This doesn't appear to be correct: function call on [L174](https://github.com/openconfig/featureprofiles/blob/9bec001da89378dd1d9d80db9493d6ba6a7b3c9b/feature/experimental/p4rt/otg_tests/google_discovery_protocol_packetout_test/google_discovery_protocol_packetout_test.go#L174) returns three different kind of packets (1 tagged,  2 untagged), and sendPackets() function sends only 1/3 of total # for each packet type

Reply:
Corrected

Reply:
thanks, lgtm